app/lib/smaxtec_api.rb

```diff
@@ -32,8 +106,11 @@ def update_sensor_readings
 
   def smaxtec_sensor_readings(sensor)
     metric = PROPERTY_MAPPING[sensor.property]
+    event = EVENT_MAPPING[sensor.property]
     unless metric
-      puts "Sensor #{sensor.name} has unrecognized property: #{sensor.property}"
+      if !event
```

Comment:
use unless

but apart from that, why do we return `nil`, if metric is nil but event is not nil?